portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,190 @@
+@import url('https://fonts.googleapis.com/css?family=Major Mono Display');
+@import url('https://fonts.googleapis.com/css?family=Codystar');
+@import url('https://fonts.googleapis.com/css?family=Nothing You Could Do');
```

Comment:
It is recommended to omit the quotation marks: https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks

Reply:
done